back/src/users/dto/update-user.input.ts

```diff
@@ -1,8 +1,34 @@
 import { CreateUserInput } from './create-user.input';
-import { InputType, Field, Int, PartialType } from '@nestjs/graphql';
-
+import { InputType, Field, PartialType } from '@nestjs/graphql';
 @InputType()
 export class UpdateUserInput extends PartialType(CreateUserInput) {
-  @Field(() => Int)
-  id: number;
+  @Field(() => String)
```

Comment:
```suggestion
  @IsNotEmpty()
  @Field(() => String)
```

저는 `@IsNotEmpty()`를 넣어줬는데 안넣어줘도 NotEmpty 처리가 되는지 궁금합니다.